src/main/java/seedu/ddd/logic/commands/AddCommand.java

```diff
@@ -1,125 +1,9 @@
 package seedu.ddd.logic.commands;
 
-import static java.util.Objects.requireNonNull;
-import static seedu.ddd.logic.parser.CliFlags.FLAG_CLIENT;
-import static seedu.ddd.logic.parser.CliFlags.FLAG_VENDOR;
-import static seedu.ddd.logic.parser.CliSyntax.PREFIX_ADDRESS;
-import static seedu.ddd.logic.parser.CliSyntax.PREFIX_EMAIL;
-import static seedu.ddd.logic.parser.CliSyntax.PREFIX_NAME;
-import static seedu.ddd.logic.parser.CliSyntax.PREFIX_PHONE;
-import static seedu.ddd.logic.parser.CliSyntax.PREFIX_SERVICE;
-import static seedu.ddd.logic.parser.CliSyntax.PREFIX_TAG;
-
-import seedu.ddd.commons.util.ToStringBuilder;
-import seedu.ddd.logic.Messages;
-import seedu.ddd.logic.commands.exceptions.CommandException;
-import seedu.ddd.model.AddressBook;
-import seedu.ddd.model.Model;
-import seedu.ddd.model.contact.client.Client;
-import seedu.ddd.model.contact.common.Contact;
-import seedu.ddd.model.contact.vendor.Vendor;
-
 /**
- * Adds a contact to the address book.
+ * This interface provides constants for add command-related keywords and flags.
  */
-public class AddCommand extends Command {
-
+public interface AddCommand {
```

Comment:
Would it be better if you make `AddCommand` an abstract class so that it can extend `Command`? I think its more logical to have `AddContactCommand extends AddCommand` instead of `AddContactCommand extends Command implements AddCommand`.

Reply:
I agree with this, so we have both `AddContactCommand` and `AddEventCommand` both extending `Command`